tracer/src/Datadog.Trace/IDatadogTracer.cs

```diff
@@ -19,6 +19,8 @@ internal interface IDatadogTracer
 
         ITraceSampler Sampler { get; }
 
+        ISpanSampler SpanSampler { get; }
```

Comment:
Have you tried renaming `Sampler` above to `TraceSampler`? I don't know how bad that would be, may not be worth it.

Reply:
Sorry forgot to reply to this one

I initially did change it early on and it ended up touching too many files, so I'll do it in a separate PR.